assets/js/components/product-preview/sad-stories/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ProductPreview from '../';
+
+/**
+This story is not working because ProductPreview depends on wcSettings, e.g. THUMBNAIL_SIZE
+I.e. it's coupled to WordPress + Woo environment.
+- Can we get wcSettings available in storybook environment?
+- Is it worthwhile having stories for components like this, or should we limit to simpler decoupled components?
+*/
```

Comment:
I'd remove this story for now.  Having storybook helps us discover coupled components that over time we can decouple.  There may also be something we can do to setup some dummy globals for the stories but that can be experimented with in followups.